rust/otap-dataflow/crates/otlp/src/grpc_stubs/mod.rs

```diff
@@ -0,0 +1,68 @@
+#[allow(missing_docs)]
+#[path = ""]
+pub mod proto {
+    #[path = ""]
+    pub mod collector {
+        #[path = ""]
+        pub mod logs {
+            #[allow(unused_qualifications)]
+            #[allow(unused_results)]
+            #[allow(clippy::enum_variant_names)]
+            #[allow(rustdoc::invalid_html_tags)]
+            #[path = "opentelemetry.proto.collector.logs.v1.rs"]
+            pub mod v1;
+        }
+        #[path = ""]
+        pub mod metrics {
+            #[allow(unused_qualifications)]
+            #[allow(unused_results)]
+            #[allow(clippy::enum_variant_names)]
+            #[allow(rustdoc::invalid_html_tags)]
+            #[path = "opentelemetry.proto.collector.metrics.v1.rs"]
+            pub mod v1;
+        }
+        #[path = ""]
+        pub mod trace {
+            #[allow(unused_qualifications)]
+            #[allow(unused_results)]
+            #[allow(clippy::enum_variant_names)]
+            #[allow(rustdoc::invalid_html_tags)]
+            #[path = "opentelemetry.proto.collector.trace.v1.rs"]
+            pub mod v1;
+        }
+    }
+
+    #[path = ""]
```

Comment:
These are more than only gRPC stubs. In the otel-arrow-rust crate we have src/proto/mod.rs, and I wonder if grpc_stubs is the right name here.